docs/playbooks/dependencies.md

```diff
@@ -105,4 +91,36 @@ ansible-playbook playbooks/dependencies/install-sls.yml
 ```
 
 !!! important
-    You must have already installed the development (pre-release) catalogs, pre-release builds are not available directly from the IBM Operator Catalog.
\ No newline at end of file
+    You must have already installed the development (pre-release) catalogs, pre-release builds are not available directly from the IBM Operator Catalog.
+
+
+## Install AppConnect
```

Comment:
Does the AppConnect occur inside the SLS? I just found out that the AppConnect doc appeared before the SLS example codes for the release and prerelease versions so I moved them to be under the SLS examples and then added the GPU install doc under that part.  Please let me know if I am wrong so that I can revert it.